### PR TITLE
brief skeleton for events page

### PR DESCRIPTION
**Pull request by mjhajharia**
check #57 

This mostly resembles the organizers page structure and only has preliminary structure

Please drop any ideas about layout changes:

- Are we ok with this or do we prefer a table instead, like [this](https://pymc-devs.github.io/pymcon/schedule)
- Are only css changes needed for size/design of cards
- Should we use hugo [taxonomies](https://jpanther.github.io/congo/docs/content-examples/#taxonomy-pages) for authors / type of event etc.
- How much information about an event do we want on

  1. Event page
  2. Overall - as in do we want pages for authors, categories or other such things.

  Or do we just prefer simple tabulation - in a .md table or card whichever form

cc: @canyon289 @purna135 @cluhmann

### CODE REVIEW

**Comment by purna135** [comment]
Can you share a preview/screensort of the current changes?


Maybe we can add the card we made for the website's initial edition. which can be found here, https://purna135.github.io/pymcon-hugo/

image

**Comment by canyon289** [comment]
> Can you share a preview/screensort of the current changes?
> 
> Maybe we can add the card we made for the website's initial edition. which can be found here, https://purna135.github.io/pymcon-hugo/
> 
> image

Using this card is an amazing idea! It has all the elements we need

File: layouts/_default/events.html [hunk 1]
```diff
@@ -0,0 +1,50 @@
+{{ define "customCSS" }}
```
Comment (canyon289) [inline layouts/_default/events.html:1]: Thanks for makign this easily extensible

**Comment by canyon289** [comment]
This is a good start. Thanks for adding the page and a configurable toml

Two questions
* Can we blend what you have with what @purna135 has on his card? Perhaps it'll be possible to copy paste that code so it isn't too much work to recreate?
* Is it possible to have an  a featured event on the front page?  Perhaps it can be a flag on each event configuration file  if its featured or not in toml and if it is it shows up on the front page We can do this in another PR if need be.

image

**Comment by canyon289** [comment]
Bigger screenshot for reference 

image

**Comment by purna135** [comment]
> Using this card is an amazing idea! It has all the elements we need. Curious how it matches up to the card in this PR.


We already have this card; I recall commenting these lines of code when we made the website public.

See PR, https://github.com/pymc-devs/pymcon_web_series/pull/152

I believe we can enable it and create a separate page for events.

**Comment by purna135** [comment]
> Can we blend what you have with what @purna135 has on his card?

If you prefer, I can uncomment the code for this card and create a separate page for the event. Because I already designed this card, it will take me less time.

**Comment by reshamas** [comment]
@purna135 Do you think we can choose a size for the card that is consistent with the Meetup size?
Such as 1200 by 630 px?

**Comment by purna135** [comment]
Are you referring to this card? Yes, we can absolutely choose this size.

image

**Comment by reshamas** [comment]
@purna135 Yes, this one, which shows the social card.

Screen Shot 2022-12-19 at 12 40 47 PM

**Comment by purna135** [comment]
Hi @mjhajharia, will you apply the changes mentioned above to this PR?

Or should I create a new PR for this?

**Comment by cluhmann** [comment]
Is there anything we can do to help this along?  We have the rough event schedule confirmed, so we can begin filling in the skeleton with actual content relatively quickly after this is merged.

**Comment by purna135** [comment]
Let me open another PR that incorporates the above-mentioned changes.

**Comment by mjhajharia** [comment]
hi! sorry for the long silence, I wasn't sure what was required because the `.toml` I added had all the attributes in the `incoming_talks` file and a few more. Anyway, I incorporated that card into the website as well. So now, you can simply add `true` or `false` to `highlighted` as an atttribute in `events.toml` to mark an event (for viewing it on the home page). Otherwise, it stays the same on the events page.


Here are (zoomed out) previews for the home page (featured events) as well as the events page.

image

image

Let me know if this works!

**Comment by canyon289** [comment]
Other event page is merged so closing this one. Thanks @mjhajharia for your contributions as well

**Pull request closed without merging**